MYSQL/tables/schema.sql

```diff
@@ -0,0 +1,36 @@
+DROP DATABASE IF EXISTS airbnb;
+CREATE DATABASE airbnb;
+use airbnb;
+
+
+CREATE TABLE suggestions(
+  id INT(20) PRIMARY KEY AUTO_INCREMENT NOT NULL,
+  listingId INT(20) NOT NULL,
+  roomtype VARCHAR(255),
+  numbOfBedrooms INT(10),
+  placeName VARCHAR(255),
+  price DECIMAL(10,2),
+  pictureURL VARCHAR(255)
+);
+
+-- CREATE TABLE roomtype(
+--   id INT(20) PRIMARY KEY NOT NULL,
+--   type VARCHAR(255)
+-- );
+
```

Comment:
what brought you to remove roomtype?

Reply:
This was for Suggestions app service. I am getting roomtype information from Eric. Pretty much all of the information
for suggestions I'll be making api request to eric and Krissy to get the data.